src/main/java/com/iexec/worker/docker/DockerService.java

```diff
@@ -229,6 +254,12 @@ public void stopRunningContainers() {
         });
     }
 
+    private static String parseRegistryAddress(String imageName) {
```

Comment:
I think we need to replace default registry address of docker lib with the one of our common lib